net48/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Newtonsoft.Json" version="13.0.3" targetFramework="net48" />
+  <package id="Portable.BouncyCastle" version="1.8.0" targetFramework="net48" />
```

Comment:
why changed it from 1.9.0 version?

Reply:
updated 

https://github.com/razorpay/razorpay-dot-net/blob/10148cd90cfb84ff83d6f4e8643488f5fe22eef5/net48/packages.config#L4